conda/environments/all_cuda-118_arch-x86_64.yaml

```diff
@@ -64,7 +64,8 @@ dependencies:
 - pandoc
 - pip
 - pre-commit
-- protobuf>=4.21.6,<4.22
+- protobuf==4.21
+- protobuf>=4.21,<5
```

Comment:
What does it mean when we list a package twice like this?

Reply:
Fixed.